pyproject.toml

```diff
@@ -57,7 +57,7 @@ documents = "documents.ext:DocumentsExt"
 oai_s3_harvester = "common.ext:OaiS3HarvesterExt"
 
 [project.entry-points."invenio_db.alembic"]
-documents = "common:alembic"
+documents = "documents:alembic"
```

Comment:
Alembic should stay in common package

Reply:
thanks for the note! it is again in common